lab_3_keywords_textrank/main.py

```diff
@@ -233,7 +275,18 @@ def add_edge(self, vertex1: int, vertex2: int) -> int:
                 0 if edge was added successfully, otherwise -1
         In case of vertex1 being equal to vertex2, -1 is returned as loops are prohibited
         """
-        pass
+        if vertex1 == vertex2:
+            return -1
+        for vertex in [vertex1, vertex2]:
```

Comment:
I hope, you can use: `for vertex in vertex1, vertex2:`

Reply:
done